docs/specs/moniker.yml

```diff
@@ -100,7 +100,7 @@ outputs:
   docs/a.json: |
     {
       "content": "<p>No version</p>",
-      "monikers": []
+      "!monikers": null
```

Comment:
would you please help to add test for empty Dictionary?

Reply:
for json.net, Dictionary is JObject